[2021-07-19 08:52:28,874] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: autobot.insert_data_to_dbb 2021-07-18T07:54:25.449895+00:00 [queued]>
[2021-07-19 08:52:29,035] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: autobot.insert_data_to_dbb 2021-07-18T07:54:25.449895+00:00 [queued]>
[2021-07-19 08:52:29,035] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-19 08:52:29,035] {taskinstance.py:1088} INFO - Starting attempt 9 of 9
[2021-07-19 08:52:29,035] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-19 08:52:29,385] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): insert_data_to_dbb> on 2021-07-18T07:54:25.449895+00:00
[2021-07-19 08:52:29,388] {standard_task_runner.py:52} INFO - Started process 2073 to run task
[2021-07-19 08:52:29,392] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'autobot', 'insert_data_to_dbb', '2021-07-18T07:54:25.449895+00:00', '--job-id', '93', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/AutoBot.py', '--cfg-path', '/tmp/tmpwhferl56', '--error-file', '/tmp/tmpp8h1gq3m']
[2021-07-19 08:52:29,392] {standard_task_runner.py:77} INFO - Job 93: Subtask insert_data_to_dbb
[2021-07-19 08:52:30,642] {logging_mixin.py:104} INFO - Running <TaskInstance: autobot.insert_data_to_dbb 2021-07-18T07:54:25.449895+00:00 [running]> on host apache-airflow.us-central1-a.c.eminent-it-info.internal
[2021-07-19 08:52:32,172] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=autobot
AIRFLOW_CTX_TASK_ID=insert_data_to_dbb
AIRFLOW_CTX_EXECUTION_DATE=2021-07-18T07:54:25.449895+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-18T07:54:25.449895+00:00
[2021-07-19 08:52:33,751] {python.py:151} INFO - Done. Returned value was: None
[2021-07-19 08:52:34,042] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=autobot, task_id=insert_data_to_dbb, execution_date=20210718T075425, start_date=20210719T085228, end_date=20210719T085234
[2021-07-19 08:52:34,759] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-19 08:52:35,322] {local_task_job.py:151} INFO - Task exited with return code 0
